### PR TITLE
[WIP] Fix simple form exception at withdraws 

### PR DESCRIPTION
**Pull request by ymasiuk**

### CODE REVIEW

File: app/controllers/private/withdraws/satoshis_controller.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 module Private::Withdraws
   class SatoshisController < ::Private::Withdraws::BaseController
     include ::Withdraws::Withdrawable
+
+    def new
```
Comment (yivo) [inline app/controllers/private/withdraws/satoshis_controller.rb]: @emasiuk you should put this in `Withdraws::Withdrawable`

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by yivo** [comment]
@emasiuk could you please check tests?

**Review by yivo — APPROVED** [review]

**Comment by yivo** [comment]
@emasiuk please rebase and run tests

**Comment by yivo** [comment]
@emasiuk I believe Peatio doesn't has fully working `/withdraws/satoshis/new`. Probably it wasn't implemented or it was abandoned. We need to strip this functionality.

**Comment by ApsOps** [comment]
@yivo Hi. Satoshi withdraws are not working for me. Is this a known issue?

**Pull request closed without merging**